docs/docs/workflows/terminate.md

```diff
@@ -29,3 +31,22 @@ Termination can fail if additional resources have been created that depend on th
 resources manually. Just run the `terminate` command again afterward to continue the termination process of the cluster.
 
 :::
+
+</tabItem>
+<tabItem value="terraform" label="Terraform">
+Terminate the cluster by running:
+
+```bash
+terraform destroy
+```
+
+Delete all files which are no longer needed:
```

Comment:
```suggestion
Delete all files that are no longer needed:
```
restrictive clause requires *that* in American English

Reply:
Interesting, I did not know that.